domain/src/main/java/org/oppia/domain/oppialogger/analytics/AnalyticsController.kt

```diff
@@ -154,4 +173,79 @@ class AnalyticsController @Inject constructor(
       )
       .build()
   }
+
+  /**
+   * Checks network connectivity of the device.
+   * Saves the [eventLog] to the [eventLogStore] in the absence of it.
+   * Uploads to remote service in the presence of it.
+   */
+  private fun checkNetworkAndManageEventLog(eventLog: EventLog) {
+    when (networkConnectionUtil.getCurrentConnectionStatus()) {
+      NONE -> addEventLog(eventLog)
+      else -> eventLogger.logEvent(eventLog)
+    }
+  }
+
+  /**
+   * Adds an event to the storage.
+   * At first, it checks if the size of the store isn't exceeding [eventLogStorageCacheSize]
```

Comment:
Here & elsewhere: either combine these sentences into a single paragraph, or use empty lines to denote different parts of the summary. It's a bit hard to read, as-is.

Reply:
Used empty lines to denote different parts.

Reply:
Resolving on behalf of Ben